scripts/qa_crowbarsetup.sh

```diff
@@ -4608,7 +4608,10 @@ function onadmin_prepare_crowbar_upgrade()
         if crowbarctl upgrade repocheck crowbar --format plain | grep "missing" ; then
             complain 11 "Some repository is missing on admin server. Cannot upgrade."
         fi
-        safely crowbarctl upgrade prepare
+        # FIXME: crowbarctl command can time out easily
```

Comment:
BTW, that crowbarctl command was at this place only for a few days before we realized it does not work. It's not like non-standard use of low level API, it's basically returning to the solution that worked before without any problem